build.gradle

```diff
@@ -17,6 +17,7 @@ ext {
 allprojects {
     repositories {
         jcenter()
+        maven { url 'https://dl.bintray.com/twofortyfouram/maven' }
```

Comment:
Yes, this is needed, the twofortyfouram library is not available from `jcenter()`